### PR TITLE
CURA-5389 Check and confirmation before Remove package 

### PR DESCRIPTION
**Pull request by jackha**
Added functions to make it able to explore package contents for above functionality in Cura.

### CODE REVIEW

File: UM/PackageManager.py [hunk 1]
```diff
@@ -402,3 +404,51 @@ def getPackageLicense(self, filename: str) -> Optional[str]:
                                             file_info.filename)
                         license_string = None
         return license_string
+
+    ##  Find the package files by package_id by looking at the installed folder
+    @staticmethod
+    def getPackageFiles(package_id) -> List[Tuple[str, List[str]]]:
+        data_storage_dir = os.path.abspath(Resources.getDataStoragePath())
```
Comment (ianpaschal) [inline UM/PackageManager.py]: Suspect issues if `Resources.getDataStoragePath()` returns `None`

File: UM/PackageManager.py [hunk 2]
```diff
@@ -402,3 +404,51 @@ def getPackageLicense(self, filename: str) -> Optional[str]:
...
+
+        os_walk = []
+        dirs_to_check = []
+        result = []  # 2-tuples of (dir, file_names)
+        for root_path, dir_names, file_names in os.walk(data_storage_dir):
```
Comment (ianpaschal) [inline UM/PackageManager.py]: Can be errors if `data_storage_dir` is `None`

File: UM/PackageManager.py (hunk 1 above)
Comment (LipuFei) [reply UM/PackageManager.py]: `Resources.getDataStoragePath()` always returns a `str`, so that's fine.

File: UM/PackageManager.py (hunk 2 above)
Comment (jackha) [reply UM/PackageManager.py]: data_storage_dir is the result of os.path.abspath and that's always a string.
os.path.abspath's input is Resources.getDataStoragePath() and that also always returns a string.

**Pull request merged**